server/models/User.ts

```diff
@@ -536,6 +537,14 @@ class User extends ParanoidModel {
         },
         transaction: options.transaction,
       }),
+      NotificationSetting.findOrCreate({
```

Comment:
Lets only create this if the user is an admin as regular users cannot export anyway it'll reduce down on unused rows being written.